src/command.h

```diff
@@ -418,6 +419,7 @@ class ClearColorCommand : public PipelineCommand {
   explicit ClearColorCommand(Pipeline* pipeline);
   ~ClearColorCommand() override;
 
+  // Colours are stored in the range 0.0-1.0
```

Comment:
Nit: `the range 0.0-1.0` -> `the range 0.0 - 1.0` (add spaces to exactly match the previous comment)

Reply:
done